Source/Common/Common.cs

```diff
@@ -44,6 +44,13 @@ public struct Waveform
         public bool IdleDurationPresent;
         public double RuntimeScaling;
         public string Script;
+
+        public void UpdateWaveformNameAndScript(string newName)
+        {
+            // Update the script with the new waveform name
+            Script = Script?.Replace(Name, newName);
+            Name = newName;
+        }

```

Comment:
how is this going to translate to lv?

Reply:
[Update Waveform Name and Script](https://github.com/NISystemsEngineering/rfic-reference-design-libraries-labview/blob/master/Source/Common/Update%20Waveform%20Name%20and%20Script.vi)

Reply:
sorry should have been more specific. you've made it a class member. i'm just wondering if we really want it to be a class member since you can't do that in lv.

Reply:
Agreed this was fine.